taxonomy/apis.py

```diff
@@ -0,0 +1,226 @@
+"""APIs for lore taxonomy application"""
+
+from taxonomy.models import Vocabulary, Term
+from learningobjects.models import LearningObject, LearningObjectType
+
+
+def get_vocabulary(vocabulary_id):
+    """
+    Lookup vocabulary given its id
+    :param vocabulary_id: Vocabulary id
```

Comment:
Unfortunately PyCharm doesn't support Google style docstrings which makes me prefer reStructuredText instead. I agree with the need for types.